ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -36,6 +36,26 @@ SD:
   - supply_air_damper_percentage_command
   implements:
   - CONTROL
+  
+EPM:
+  description: "Basic Electrical parameter monitoring."
+  is_abstract: true
+  uses:
+  - power_status
+  - voltage_sensor
+  - current_sensor
+  - power_sensor
+  - energy_accumulator
+  implements:
+  - MONITORING
+
+RDM:
```

Comment:
Not used elsewhere. Remove.

Reply:
EPM abstracts removed from the current pull request...